package.json

```diff
@@ -16,13 +16,13 @@
   "scripts": {
     "start": "electron .",
     "debug": "node-inspector & electron --debug=5858 . & xdg-open http://127.0.0.1:8080/debug?ws=127.0.0.1:8080&port=5858",
-    "get-siad": "npm run sia-repo && npm run sia-master && npm run sia-import",
+    "postinstall": "npm run sia-repo && npm run sia-master && npm run sia-import",
     "clean": "rm -rf build node_modules app/dependencies app/config.json **/*.swp npm-debug.log",
     "fresh": "npm run clean && npm install && npm start",
     "lint": "jshint . --verbose --exclude-path=.jshintignore",
     "doc": "jsdoc -c .jsdocrc",
     "sia-repo": "test -d $GOPATH/src/github.com/NebulousLabs/Sia || go get -u github.com/NebulousLabs/Sia/...",
-    "sia-master": "(cd $GOPATH/src/github.com/NebulousLabs/Sia && git fetch && git pull && git checkout v0.3.3.3-beta && make dependencies && make)",
+    "sia-master": "(cd $GOPATH/src/github.com/NebulousLabs/Sia && git fetch && git pull && git checkout master && make dependencies && make)",
```

Comment:
I forget. I think it's a good command, just not sure how easy it will be to stay up to date in the near future.